pkgs/by-name/ip/ipbus-uhal/package.nix

```diff
@@ -0,0 +1,70 @@
+{
+  lib,
+  stdenv,
+  bash,
+  cacert,
+  boost,
+  pugixml,
+  python3,
+  python3Packages,
+  fetchFromGitHub,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "uhal";
+  version = "2.8.16";
+  src = fetchFromGitHub {
+    owner = "ipbus";
+    repo = "ipbus-software";
+    hash = "sha256-R+a9VmONyWh3BEYoMjRcXKv+3HaNcKbJDnYH1hXHdPg=";
+    rev = "refs/tags/v${version}";
+  };
+
+  patches = [
+    ./avoid-bin-bash.patch
+  ];
+
+  nativeBuildInputs = [
+    cacert
+    python3
+    python3.pkgs.distutils
+  ];
+  buildInputs = [
+    boost
+    pugixml
+    python3.pkgs.distutils
+    python3.pkgs.pybind11
```

Comment:
Are these still buildInputs? (I don't think `distutils` is a buildInput)

Reply:
got rid of.